components/dashboard/src/components/PendingChangesDropdown.tsx

```diff
@@ -38,16 +37,15 @@ export default function PendingChangesDropdown(props: { workspaceInstance?: Work
         }
     }
     if (totalChanges <= 0) {
-        return <div className="text-sm text-gray-400 dark:text-gray-500">No Changes</div>;
+        return <></>;
```

Comment:
**thought**: Removing the unnecessary state of _No Changes_. Once NextGen workspaces or experiments like https://github.com/gitpod-io/gitpod/pull/18300 land, this would also surface the real time information of git status